sdk/state/open.go

```diff
@@ -84,6 +84,15 @@ type OpenTransactions struct {
 	Formation       *txnbuild.Transaction
 }
 
+func (ot OpenTransactions) CloseTransactions() CloseTransactions {
```

Comment:
❓ should these be non-exported (right word?). Probably doesn't matter because someone can access these values from the CloseAgreement() method. But it would make it obvious that these methods are intended for internal use only

Reply:
I think it's okay if they're exported. We're only exposing data that is already exported. Actually, in #310 I removed this function 😬. Sorry.